src/Symfony/Component/Config/Definition/Builder/ArrayNodeDefinition.php

```diff
@@ -80,6 +80,76 @@ public function prototype($type)
     }
 
     /**
+     * Shorthand for prototype('variable').
+     *
+     * @return VariableNodeDefinition
+     */
+    public function prototypeVariable()
+    {
+        return $this->prototype('variable');
+    }
+
+    /**
+     * Shorthand for prototype('scalar').
+     *
+     * @return ScalarNodeDefinition
+     */
+    public function prototypeScalar()
```

Comment:
Yes you're right. That's not a big issue. Most IDE autocomplete are clever enough to show methods containing rather than starting with a string.